src/components/Project/Sidebar/index.vue

```diff
@@ -136,11 +136,23 @@
         this.$router.push({ name: "Login" })
       },
       selectNode(data) {
+        let path
         switch (data.type) {
           case "query":
             this.openTab(data)
+            path = `${data.parent}/${data.connection}/${data.name}`
+            // appProjects.toggleQueryDialog()
+            appProjects.setEditNode(path)
```

Comment:
Remove this code (Line 143 to 145). Always run the application after resolving conflicts.